src/sdk/resources/policies.ts

```diff
@@ -1,9 +1,14 @@
 import { IHttpClient } from '../http';
 import { PagedResource } from './resource';
 import { Policy } from '../models';
+import { PolicyItemTrackingGroupsResource } from './policyItemTrackingGroups';
 
 export class PoliciesResource extends PagedResource<Policy> {
 	constructor(httpClient: IHttpClient) {
 		super(httpClient, 'policy', 'policy', 'policies', 'policies', 'policies');
 	}
+
+	trackingGroups(policyId: string): PolicyItemTrackingGroupsResource {
```

Comment:
This can be removed.

Reply:
I was using this to set the text on the `Tracking group` column on the items grid. Otherwise, I'd have the tracking group id on the item, right?

Reply:
The item will have a `group.id` and `group.name` attached to the item so you can grab the text from that.